docs/blog/gatsby-and-strapi.md

```diff
@@ -0,0 +1,502 @@
+
+const makeRequest = (graphql, request) => new Promise((resolve, reject) => {
+  // Query for article nodes to use in creating pages.
+  resolve(
+    graphql(request).then(result => {
+      if (result.errors) {
+        reject(result.errors)
+      }
+
+      return result;
+    })
+  )
+});
+
+
+// Implement the Gatsby API “createPages”. This is called once the
+// data layer is bootstrapped to let plugins create pages from data.
+exports.createPages = ({ boundActionCreators, graphql }) => {
+  const { createPage } = boundActionCreators;
+
+  const getArticles = makeRequest(graphql, `
+    {
+      allStrapiArticle {
+        edges {
+          node {
+            id
+          }
+        }
+      }
+    }
+    `).then(result => {
+    // Create pages for each article.
+    result.data.allStrapiArticle.edges.forEach(({ node }) => {
+      createPage({
+        path: `/${node.id}`,
+        component: path.resolve(`src/templates/article.js`),
+        context: {
+          id: node.id,
+        },
+      })
+    })
+  });
+
+  const getAuthors = makeRequest(graphql, `
+    {
+      allStrapiUser {
+        edges {
+          node {
+            id
+          }
+        }
+      }
+    }
+    `).then(result => {
+    // Create pages for each user.
+    result.data.allStrapiUser.edges.forEach(({ node }) => {
+      createPage({
+        path: `/authors/${node.id}`,
+        component: path.resolve(`src/templates/user.js`),
+        context: {
+          id: node.id,
+        },
+      })
+    })
+  });
+
+  // Queries for articles and authors nodes to use in creating pages.
+  return Promise.all([
+    getArticles,
+    getAuthors,
+  ])
+};
+```
+
+Finally, restart the server and visit the author page from the article view's links.
+
+![Tutorial](/content/images/2018/01/Screen-Shot-2018-01-17-at-21.27.47.png)
+
+## Conclusion
+
+Congrats! You’ve successfully built a super fast and easy-to-maintain blog!
+
+Since the content is managed by Strapi, the authors can write article through a nice UI and developers only have to rebuilt the Gatsby blog in order to update the content.
+
+***Where to go next?***
+
+Feel free to continue this project to discover both Gatsby and Strapi advantages. Here are some features you can add: list of authors, article's categories, and comment system with the Strapi API or Disqus. You can also create other kind of websites (e-commerce shop, corporate website, etc.).
+
+When your project is achieved, you will probably want to deploy it. The static website generated by Gatsby can [easily be published on storage providers](https://www.gatsbyjs.org/docs/deploy-gatsby/): Netlify, S3/Cloudfront, GitHub pages, GitLab pages, Heroku, etc. Since it is a Progressive Web App without server-side rendering, we strongly advise you to use a pre-rendering solution, such as Prerender.io. The Strapi API is nothing else than a simple Node.js API, so it can be hosted on Heroku or any Linux instance that has Node.js installed.
```

Comment:
We need to remove the "Since it is a Progressive Web..." sentence since that's incorrect.